### PR TITLE
Jg/themes page updates

### LINKED ISSUE #6623: Update public website with better theming information

### PR DESCRIPTION
**Pull request by JasonGore**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #6623 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Implement changes described in #6623:

themes

colors

### CODE REVIEW

File: apps/fabric-website-resources/src/components/pages/ColorsPage.tsx [hunk 1]
```diff
@@ -379,26 +383,47 @@ export class ThemeGeneratorPage extends BaseComponent<{}, IThemeGeneratorPageSta
...
+            </PivotItem>
+            <PivotItem linkText="JSON">
+              <textarea
+                readOnly={true}
+                spellCheck={false}
+                value={JSON.stringify(ThemeGenerator.getThemeAsJson(abridgedTheme), void 0, 2)}
+                style={{ width: '100%' }}
+              />
+            </PivotItem>
+            <PivotItem linkText="Sass">
```
Comment (phkuo) [inline apps/fabric-website-resources/src/components/pages/ColorsPage.tsx]: Weren't we going to remove the Sass section altogether?

File: apps/fabric-website-resources/src/components/pages/ColorsPage.tsx [hunk 2]
```diff
@@ -379,26 +383,47 @@ export class ThemeGeneratorPage extends BaseComponent<{}, IThemeGeneratorPageSta
...
+              <p>
+                This code block initializes the theme you have configured above and loads it using the loadTheme utility function. Calling
+                loadTheme will automatically apply the configured theming to any Fabric controls used within the same app.
+              </p>
+              <textarea
+                readOnly={true}
+                spellCheck={false}
+                value={codeHeader + themeAsCode}
+                // value={codepenSamples}
+                style={{ width: '100%' }}
```
Comment (phkuo) [inline apps/fabric-website-resources/src/components/pages/ColorsPage.tsx]: IMO it looks nicer to have width:300px so it doesn't have all the extra whitespace inside the textbox. (If keeping the Sass section, it needs to be 500px instead.)

File: apps/fabric-website-resources/src/components/pages/ColorsPage.tsx [hunk 3]
```diff
@@ -379,26 +383,47 @@ export class ThemeGeneratorPage extends BaseComponent<{}, IThemeGeneratorPageSta
...
+            <PivotItem linkText="Code">
+              <p>
+                This code block initializes the theme you have configured above and loads it using the loadTheme utility function. Calling
+                loadTheme will automatically apply the configured theming to any Fabric controls used within the same app.
+              </p>
+              <textarea
+                readOnly={true}
+                spellCheck={false}
+                value={codeHeader + themeAsCode}
+                // value={codepenSamples}
```
Comment (phkuo) [inline apps/fabric-website-resources/src/components/pages/ColorsPage.tsx]: remove commented code

File: packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md [hunk 4]
```diff
@@ -1,33 +1,38 @@
 The entire color palette of the controls are themeable. We provide a set of sensible defaults, but you can override all colors individually.
+In order to do this you must provide a theme at app startup and provide it to `loadTheme()`.
 
-## Loading Themes
+## Using Themes
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md]: Hm, "invoking" themes?

File: packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md [hunk 5]
```diff
@@ -1,33 +1,38 @@
 The entire color palette of the controls are themeable. We provide a set of sensible defaults, but you can override all colors individually.
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md]: "Themable" without the E in the middle.

Also, we should have a link on this page to the theme generator.

File: apps/fabric-website/src/pages/Customizations/ColorsCustomizationPage.tsx [hunk 6]
```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
-import { PageHeader } from '../../../components/PageHeader/PageHeader';
-import { ThemeGeneratorPage } from '@uifabric/fabric-website-resources/lib/components/pages/ThemeGeneratorPage';
-const pageStyles: any = require('../../PageStyles.module.scss');
+import { PageHeader } from '../../components/PageHeader/PageHeader';
+import { ColorsPage } from '@uifabric/fabric-website-resources/lib/components/pages/ColorsPage';
```
Comment (phkuo) [inline apps/fabric-website/src/pages/Customizations/ColorsCustomizationPage.tsx]: This is not a "Colors" page though, the page specifically generates themes and does nothing else, not even documentation.

Comment (dzearing) [reply apps/fabric-website/src/pages/Customizations/ColorsCustomizationPage.tsx]: Hmm maybe so. Brainstorm in the morning on this.

We could have a page later whoch describes what palette is and what it contains.

Comment (Jahnp) [reply apps/fabric-website/src/pages/Customizations/ColorsCustomizationPage.tsx]: Agreed, there's already a page for Colors under styles. What about `ColorsCustomizationsPage`?

Comment (Jahnp) [reply apps/fabric-website/src/pages/Customizations/ColorsCustomizationPage.tsx]: OMG nevermind I didn't read down to the next line 😂 My bad.

File: packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md (hunk 5 above)
Comment (Jahnp) [reply packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md]: Not ideal, since it's a little more optimized for editing source vs just viewing, but we could create an aka.ms link with a link to its absolute GitHub link, e.g. https://github.com/OfficeDev/office-ui-fabric-react/edit/master/packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md. 

We do this for "Edit" links to control documentation:
image

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md]: I think we want to do a link *from* the MD to the color customization page (https://developer.microsoft.com/en-us/fabric#/styles/themegenerator). Are there aka.ms links for any of the developer pages?

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md]: Also, I don't think "themable" is a word. 😆   http://www.yourdictionary.com/themeable

Comment (phkuo) [reply packages/office-ui-fabric-react/src/components/Theme/docs/ThemesOverview.md]: @JasonGore we've been using "themable" for a long long time now:
https://docs.microsoft.com/en-us/sharepoint/dev/general-development/how-to-make-custom-css-files-themable-in-sharepoint

Bing gets more hits on "themEable" than "themable", but Google is the opposite.
And .NET uses "themEable" instead.
https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.themeableattribute?view=netframework-4.7.2

In summary, I have no idea what to do =)
We should just pick one and stick to it.
